generator.py

```diff
@@ -127,13 +129,14 @@ def extract_table_data(courses):
                 start = int(start*2) # columns are half-hours, not hours
                 for day in days.split():
                     print(course['course_id'], instructor, session, day, start/2, room)
+                    instructorLast = instructor.split()[-1]
                     room_cell = {
-                            'label': instructor,
+                            'label': instructorLast,
```

Comment:
Your style should be consistent with the rest of the document. The common practice for Python is underscores, not camelCase.

Reply:
fixed